flake.nix

```
@@ -52,8 +52,8 @@
         hevm = pkgs: pkgs.lib.pipe ((hsPkgs pkgs).callCabal2nix "hevm" (pkgs.fetchFromGitHub {
           owner = "ethereum";
           repo = "hevm";
-          rev = "9924e80dd0226c1f3e1eda651bbeb3558f419843";
-          sha256 = "sha256-HGvWD7AjqKinE/oYu9L4b2HetcKrO3B+hNCzJ761XAk=";
+          rev = "e180f31b8895ed8bd69ff06fdc081bc1a5ec0ce5";
+          sha256 = "sha256-CXZI3/8B1bKceEf2pRFlqAM4/gRbZLtwqw8wyqe/VBY=";
```

```suggestion
          rev = "7fa6a959bea935f476100037aa84531523cd9d8a";
          sha256 = "sha256-4dTXB/3F3180L5tBhDVky2A6irbGDN4rZHdnDdc+j64=";
```

Matching the `update hevm` PR in echidna